yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -0,0 +1,584 @@
+      type empty;
+      configd:help "Log changes to the IS-IS adjacencies in this area.";
+      description "Log changes to the IS-IS adjacencies in this area.";
+    }
+    container lsp-gen-interval {
+      configd:help "Minimum LSP regeneration interval.";
+      description "Minimum LSP regeneration interval.";
+      leaf level-1 {
+        type uint16 {
+          range "1..120";
+        }
+        units "seconds";
+        configd:help "Minimum time allowed before level-1 LSP retransmissions.";
+        description "Minimum time allowed before level-1 LSP retransmissions.";
+      }
+      leaf level-2 {
+        type uint16 {
+          range "1..120";
+        }
+        units "seconds";
+        configd:help "Minimum time allowed before level-2 LSP retransmissions.";
+        description "Minimum time allowed before level-2 LSP retransmissions.";
+      }
+    }
+    container lsp-refresh-interval {
+      description "LSP refresh interval";
+      configd:help "LSP refresh interval";
+      leaf level-1 {
+        type uint16 {
+          range "1..65535";
+        }
+        units "seconds";
+        configd:help "LSP refresh interval for level-1.";
+        description "LSP refresh interval for level-1.";
+        must ". > ../../lsp-gen-interval/level-1" {
+          error-message "lsp refresh interval must be greater than lsp general interval";
+        }
+      }
+      leaf level-2 {
+        type uint16 {
+          range "1..65535";
+        }
+        units "seconds";
+        configd:help "LSP refresh interval for level-2.";
+        description "LSP refresh interval for level-2.";
+        must ". > ../../lsp-gen-interval/level-2" {
+          error-message "lsp refresh interval must be greater than lsp general interval";
+        }
+      }
+    }
+
+    container max-lsp-lifetime {
+      configd:help "Maximum LSP lifetime.";
+      description "Maximum LSP lifetime.";
+      leaf level-1 {
+        type uint16 {
+          range "350..65535";
+        }
+        units "seconds";
+        configd:help "Maximum LSP lifetime for level-1.";
+        description "Maximum LSP lifetime for level-1.";
+      }
+      leaf level-2 {
+        type uint16 {
+          range "350..65535";
+        }
+        units "seconds";
+        configd:help "Maximum LSP lifetime for level-2.";
+        description "Maximum LSP lifetime for level-2.";
+      }
+    }
+
+    leaf set-overload-bit {
+      type empty;
+      configd:help "If true, avoid any transit traffic.";
+      description "If true, avoid any transit traffic.";
+    }
+
+    container spf-delay-ietf {
+      presence "Present if IETF SPF back-off delay is enabled.";
+      configd:help "IETF SPF delay algorithm.";
+      description "IETF SPF delay algorithm.";
+      leaf init-delay {
+        configd:help "Delay used while in QUIET state";
+        description "Delay used while in QUIET state";
+        default 500;	
+        type uint16 {
```

Comment:
You could usefully define some common types for delays and timers as this will make the YANG more concise and more readable (and ensure consistency across different nodes)

Reply:
This is resolved. 

Thanks,